build/phase.go

```diff
@@ -129,8 +129,8 @@ func (p *Phase) Run(context context.Context) error {
 		context,
 		p.docker,
 		p.ctr.ID,
-		p.logger.VerboseWriter().WithPrefix(p.name),
-		p.logger.VerboseErrorWriter().WithPrefix(p.name),
+		logging.NewPrefixWriter(p.logger.Writer()).WithPrefix(p.name),
```

Comment:
Wonder if the prefix should just be part of the constructor now that there's a constructor, rather than a separate method